csharp/Facebook.CSSLayout/Native.cs

```diff
@@ -0,0 +1,263 @@
+﻿/**
+ * Copyright (c) 2014-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+using System;
+using System.Runtime.InteropServices;
+
+namespace Facebook.CSSLayout
+{
+    internal static class Native
+    {
+        private const string DllName = "CSSLayout.dll";
+
+        [DllImport(DllName)]
```

Comment:
I think this should omit the `.dll` suffix so it could potentially work on Linux in the future.